src/legacy/core_plugins/kibana/server/tutorials/apm/envs/on_prem.js

```diff
@@ -83,17 +89,16 @@ export function onPremInstructions(apmIndexPattern) {
             defaultMessage: 'You have correctly setup APM Server',
           }),
           error: i18n.translate('kbn.server.tutorials.apm.apmServer.statusCheck.errorMessage', {
-            defaultMessage: 'APM Server has still not connected to Elasticsearch',
+            defaultMessage: 'No APM Server detected. Please make sure it is running and you have updated to 7.0 or higher.',
           }),
           esHitsCheck: {
             index: apmIndexPattern,
             query: {
               bool: {
-                filter: {
-                  exists: {
-                    field: 'observer.listening',
-                  },
-                },
+                filter: [
+                  { exists: { field: 'observer.listening' } },
```

Comment:
I think it would be better for onboarding doc queries to filter on processor.name === "onboarding", to be clearer.

Reply:
Good catch 🎣